README.md

```diff
@@ -34,19 +35,27 @@ Lastly, a Dockerfile is provided for installing pre-requisite packages & launchi
 
 Please ensure you meet the [pre-requisites](https://rapids.ai/start.html#prerequisites)
 
-Our container extends the base notebooks container. To build:
+Our container extends the base notebooks container. Unlike our Notebooks repo, which comes with the container, Notebooks Extended is meant to be a constatly updating source of community contributions.  We've made it easy to include Notebooks Extended in your RAPIDS container with 3 easy steps:
+
+Step 1: Download your RAPIDS container
+```bash
+docker pull rapidsai/rapidsai:latest
+```
+
+Step 2: Git Pull Notebooks Extended into the folder of your choice (change "#/folder/of/your/choice" into wherever you desire Notebooks Extended to be). 
+
 ```bash
+mkdir #/folder/of/your/choice
 git clone https://github.com/rapidsai/notebooks-extended
-cd notebooks-extended
-docker build -t notebooks-extended .
 ```
 
-To run:
+Step 3: Run Docker mounting your Notebooks Extended folder as a volume, changing "/folder/of/your/choice/" to where you put Notebooks Extended
 ```bash
-docker run -p 8888:8888 -it notebooks-extended
+docker run --runtime=nvidia --rm -it -p 8888:8888 -p 8787:8787 -p 8786:8786 -v /folder/of/your/choice/ -it rapidsai/rapidsai:latest
 ```
```

Comment:
I think you need `utils/start-jupyter.sh` after entering the container too.

Reply:
you're right, i should include that.